Documentation/source/Event tracking guide.md

```diff
@@ -0,0 +1,30 @@
+Event Tracking
+=============================
+
+The version 5.2 of Gini Vision Library introduces the ability to track user events in the Screen API. In order to receive the events, implement the `GiniVisionTrackingDelegate` protocol and supply the delegate when initializing GVL. For example:
+
+```swift
+let viewController = GiniVision.viewController(withClient: client,
+                                               importedDocuments: visionDocuments,
+                                               configuration: visionConfiguration,
+                                               resultsDelegate: self,
+                                               documentMetadata: documentMetadata,
+                                               trackingDelegate: trackingDelegate)
```

Comment:
You should also show how to add the tracking delegates when using the Component API.